src/Modules/Unix/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1

```diff
@@ -24,7 +24,6 @@ CmdletsToExport= "Format-List", "Format-Custom", "Format-Table", "Format-Wide",
     "Get-Runspace", "Debug-Runspace", "Enable-RunspaceDebug", "Disable-RunspaceDebug",
     "Get-RunspaceDebug", "Wait-Debugger" , "Get-Uptime", "New-TemporaryFile", "Get-Verb", "Format-Hex"
 FunctionsToExport= "Import-PowerShellDataFile"
-AliasesToExport= "fhx"

```

Comment:
If alias is not declared in `.psd1` file, using the alias when the module hasn't been loaded will not trigger the module autoloading. Try the following and you will see it:
```
gmo | rmo
gin
```